packages/vscode-extension/messages.json

```diff
@@ -54,6 +54,8 @@
 	"ERRORS_NO_ACTIVE_FILE_OR_WORKSPACE": "There is no active file or the active file is not in a workspace.",
 	"ERRORS_SDK_JAVA_VERSION_NOT_COMPATIBLE": "Your Java version {0} is not compatible with SuiteCloud Extension for Visual Studio Code. Install Oracle JDK version {1}.",
 	"ERRORS_SDK_JAVA_VERSION_NOT_INSTALLED": "Java is not installed on this machine. Install Oracle JDK version {0} to run SuiteCloud Extension for Visual Studio Code.",
+	"ERRORS_SDF_OBJECT_MUST_BE_IN_OBJECTS_FOLDER": "The SDF object file should be in the Objects folder.",
```

Comment:
Could we add a reference to the command?
I would like to see something similar to `The SDF object file to update should be in the Objects folder.` or add some kind of reference to the command triggered.

Reply:
I like it! Let's edit the message to what you proposed.

Reply:
Updated.